src/app/geo/init-map.directive.js

```diff
@@ -99,6 +99,15 @@ function rvInitMap($rootScope, configService, geoService, events, referenceServi
                 return $compile(html)($rootScope);
             };
 
```

Comment:
Do we need two compile methods? I'm ok if we remove the above `$` compile function in favor of yours. If so please update plugins.

Reply:
I wasn't sure of all the locations using the compile function without scope but it's probably ok to change it.
Maybe keep it as `$` and add a flag for whether or not to use scope so existing functionality isn't changed?

Reply:
Hmm ok, merge both into one and name it `$compile`.

```js
apiMap.$compile = function(html: JQuery<HTMLElement>, useIsolatedScope: boolean = true) {
    const scope = $rootScope.$new(useIsolatedScope);
    $compile(html)(scope);
    return scope;
}
``` 
It's relatively new and not widely used yet. 

Places that need an update
https://github.com/fgpv-vpgf/fgpv-vpgf/blob/v3/api/src/panel.ts#L195
https://github.com/fgpv-vpgf/fgpv-vpgf/blob/v3/api/src/panel.ts#L266
https://github.com/fgpv-vpgf/fgpv-vpgf/blob/v3/docs/developer/plugins.md

Instead of this

```js
this.elementAttr = this.panel.map.$($(element)[0]);
```

Do this

```js
this.elementAttr = $(element); // normalizes to a JQuery<HTMLElement>
this.panel.map.$compile(element[0]); // compiles the native DOM element in the above jQuery object.
```